lib/awful/widget/layoutlist.lua

```diff
@@ -0,0 +1,151 @@
+----------------------------------------------------------------------------
+--- Display the available client layouts for a screen.
+--
+--@DOC_awful_widget_layoutlist_popup_EXAMPLE@
+--
+--@DOC_awful_widget_layoutlist_bar_EXAMPLE@
+--
+-- @author Emmanuel Lepage Vallee &lt;elv1313@gmail.com&gt;
+-- @copyright 2017 Emmanuel Lepage Vallee
+-- @classmod awful.widget.layoutlist
+----------------------------------------------------------------------------
+
+local wibox    = require("wibox")
+local awcommon = require("awful.widget.common")
+local abutton  = require("awful.button")
+local gtable   = require("gears.table")
+local beautiful= require("beautiful")
+local alayout  = require("awful.layout")
+local surface  = require("gears.surface")
+
+local module = {}
+
+local default_buttons = gtable.join(
+    abutton({ }, 1, function(a) a.callback() end)
+)
+
+local function wb_label(item)
+    local selected = alayout.get(item.screen) == item.layout
+    return item.layout.name, selected and beautiful.bg_focus or nil, nil, item.icon
```

Comment:
Shouldn't there be a `beautiful.bg_layoutlist` in here?

Reply:
done (doubling the file size... whatever)